test/PidTest.cpp

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file PidTest.cpp
+ * @brief PidGainクラス,Pidクラスをテストする
+ * @author kawano
+ */
+
+#include "Pid.h"
+#include <gtest/gtest.h>
+
+namespace etrobocon2022_test
+{
+  TEST(PidGainTest, gain)
+  {
+    double expected_p = 0.1;
+    double expected_i = 0.03;
+    double expected_d = 0.612;
+    PidGain actualPidGain(expected_p, expected_i, expected_d);
+    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);
+  }
+
+  TEST(PidGainTest, gain_zero)
+  {
+    double expected_p = 0.0;
+    double expected_i = 0.0;
+    double expected_d = 0.0;
+    PidGain actualPidGain(expected_p, expected_i, expected_d);
+    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);
+  }
+
+  TEST(PidGainTest, gain_minus)
+  {
+    double expected_p = -0.5;
+    double expected_i = -0.2;
+    double expected_d = -0.3;
+    PidGain actualPidGain(expected_p, expected_i, expected_d);
+    EXPECT_DOUBLE_EQ(expected_p, actualPidGain.kp);
+    EXPECT_DOUBLE_EQ(expected_i, actualPidGain.ki);
+    EXPECT_DOUBLE_EQ(expected_d, actualPidGain.kd);
+  }
+
+  TEST(PidTest, caluculatePid)
+  {
+    static const double DELTA = 0.01;

```

Comment:
staticは何で付けてる？
ここだけで定数だし，スコープは極力狭くしないと名前の衝突が起きると思う

あと，基本的にC++ではconstじゃなくてconstexprを使った方がいい[（参考資料）](https://qiita.com/saltheads/items/dd65935878a0901fe9e7)
```suggestion
    constexpr double DELTA = 0.01;
```